src/injectable/electron-mocks.js

```diff
@@ -83,17 +102,31 @@ module.exports = function install (config, testPageUrl) {
 
             ipc.sendInjectingStatus({ completed: true });
 
-            BrowserWindow.prototype.loadURL = origLoadURL;
+            WebContents.prototype.loadURL = origLoadURL;
 
             url = testPageUrl;
 
             windowHandler.window = this;
 
             if (config.openDevTools)
-                this.webContents.openDevTools();
+                this.openDevTools();
         }
 
-        return origLoadURL.call(this, url);
+        return origLoadURL.call(this, url, options);
+    };
+
+    WebContents.prototype.loadFile = function (filePath, options = {}) {
+        const { query, search, hash } = options;
+        const resolvedURL             = path.resolve(getCustomAppPathDir() || app.getAppPath(), filePath);
```

Comment:
We definitely should override `getAppPath`. App scripts can use it for loading resources and requiring modules. It can break things if we don't provide a correct value when a custom app path is specified.